storage.go

```diff
@@ -484,6 +497,46 @@ func (s *Storage) write(ctx context.Context, path string, r io.Reader, size int6
 	return size, nil
 }
 
+func (s *Storage) writeAppend(ctx context.Context, o *Object, r io.Reader, size int64, opt pairStorageWriteAppend) (n int64, err error) {
+	rp := o.GetID()
+
```

Comment:
Check `IsAppend` here.

Reply:
Is `isRead` also a necessary check?

Reply:
No, we don't need to check `IsRead` during `writeAppend` operation.